### ISSUE TITLE
Weird extra stuff from the PinboardLogger

### ISSUE THREAD

**Issue by dmpohlmann** [ttscoff/Slogger#424]
Hi guys

PinboardLogger is working but is inserting some extra stuff into Dayone under each bookmark.

Example:

```
<a href='dayone2://filter?tag=adhd'>#adhd</a>
```

Any ideas?

DM

**Comment by ttscoff** [comment]
That looks like something Day One is adding when it imports… 
converting the tags that Slogger adds to internal links automatically.

-Brett

**Comment by dmpohlmann** [comment]
Yep cool ok. So by that I assume that we can't do anything about it at this stage, it is a Day One issue? If so, worth me logging a ticket with them?

DM

**Comment by ttscoff** [comment]
I'm not positive and haven't taken a close look at it yet, but I assume it to be a day one issue. I probably won't have time to examine it myself for a while, so it might be worth going ahead and filing the report with them to see what they say.

- Brett
